drbd/packages.sls

```diff
@@ -12,6 +12,17 @@ install-ha-ha_sles-pattern-for-drbd:
     - fromrepo: {{ repo }}
     - pkgs:
       - patterns-ha-ha_sles
+    - retry:
+        attempts: 3
+        interval: 15
+
+install_nfs_server_for_drbd:
+  pkg.installed:
+    - pkgs:
+      - nfs-kernel-server
```

Comment:
I intend to not including `nfs` related things in drbd formula and leave the pure DRBD only things here. 
I think it is better to move it to HA/SAP/Terraform.
Also add the `state` here may incomplete when not use pattern plus `drbd.with_ha == true`. (the nfs-server is recommend in `nfs-formula`)

Sidenote: Though the vip in pillar is more like a HA thing, but it is also per DRBD resource parameter

Reply:
@arbulu89 I add the `nfs-kernel-server` to terraform `drbd_node`. Refer to https://github.com/SUSE/ha-sap-terraform-deployments/pull/239/commits/e28bc1e32ff50823cbd4dd3d008bdd978e112621